azure-devops/projects/io-pay-projects/secrets.tf

```diff
@@ -0,0 +1,29 @@
+provider "azurerm" {
+  features {}
+}
+
+variable "secrets" {
+  default = [
+    "DANGER-GITHUB-API-TOKEN",
+    "io-azure-devops-github-ro-TOKEN",
+    "io-azure-devops-github-rw-TOKEN",
+    "io-azure-devops-github-pr-TOKEN",
+    "io-azure-devops-github-EMAIL",
+    "io-azure-devops-github-USERNAME",
+    "pagopa-npm-bot-TOKEN",
```

Comment:
```suggestion
```
@infantesimone do u need to connect to npm registry to publish something? if u don't need it delete this please